cmd/ipfs-cluster-service/daemon.go

```diff
@@ -47,6 +47,10 @@ func parseBootstraps(flagVal []string) (bootstraps []ma.Multiaddr) {
 
 // Runs the cluster peer
 func daemon(c *cli.Context) error {
+	if c.String("consensus") == "" {
+		checkErr("starting daemon", errors.New("--consensus flag must be set to \"raft\" or \"crdt\""))
```

Comment:
This would fail anyways later down, but I prefer to give a very clear error message.